test/e2e/const.go

```diff
@@ -202,10 +206,11 @@ const (
 
 	AuthSecretName = "basic-auth-secret"
 
-	ShortTestLabel = "short"
-	FullTestLabel  = "full"
-	DontRunLabel   = "dontrun"
-	LocalTestLabel = "local"
+	ShortTestLabel   = "short"
+	FullTestLabel    = "full"
+	DontRunLabel     = "dontrun"
+	LocalTestLabel   = "local"
```

Comment:
I recall this was introduced by me in https://github.com/rancher/turtles/pull/187/files#diff-f41624bd6051a35a1694431afebcf5012b89a1cb9ac856464228f797065d34b1R125 while adding CAPV e2e tests but looking at the current code, that label and some more [logic](https://github.com/rancher/turtles/pull/187/files#diff-804cacacd09be55e4060407719307c3a82878d91b7d457fa71ad73ac767eb9a8R237-R239) looks redundant to me. Do we want to cleanup those as part of the same PR?

Reply:
Well spotted, I've now removed that label and associated code.